src/training/calculatorreview/Main.java

```diff
@@ -4,9 +4,79 @@
 
 public class Main {
 
+    private static Scanner in;
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+
+        in = new Scanner(System.in);
+
+        while (true) {
+            performOneCalculation();
+        }
+    }
+
+    private static void performOneCalculation() {
+        String op = askForOperator();
+        int[] n = askForNumberArray(op);
+        int a = calculateAnswer(op, n);
+
+        System.out.println("The answer is: " + a);
+        System.out.println();
+    }
+
+    private static void printWelcomeMessage() {
         System.out.println("Welcome to the calculator!");
         System.out.println("==========================");
-        new Scanner(System.in);
+    }
+
+    private static String askForOperator() {
+        String[] allowedOperators = { "+", "-", "*", "/" };

```

Comment:
We should check the input against this allowed operators variable so we know the user has entered something we expected.